src/intl/en/learn-quizzes.json

```
@@ -613,5 +613,50 @@
   "defi-5-c-label": "Bitcoin",
   "defi-5-c-explanation": "This is incorrect. Bitcoin is a simple network for storing value, not for running advanced programs. DeFi requires a more flexible system, like Ethereum, that can run complex programs to handle loans and trades automatically.",
   "defi-5-d-label": "Traditional financial institutions",
-  "defi-5-d-explanation": "This is incorrect. DeFi apps don’t need traditional financial institutions. They use blockchain programs called smart contracts to handle transactions automatically."
+  "defi-5-d-explanation": "This is incorrect. DeFi apps don’t need traditional financial institutions. They use blockchain programs called smart contracts to handle transactions automatically.",
+  "smart-contracts-1-prompt": "How are smart contracts characterized?",
+  "smart-contracts-1-a-label": "Smart contracts are just like real (paper) contracts, but stored digitally on the blockchain to save the content securely.",
+  "smart-contracts-1-a-explanation": "Smart contracts use a similar logic to traditional contracts, but otherwise have little in common.",
+  "smart-contracts-1-b-label": "They are linked to an AI system, which allows them to make decisions that trigger blockchain transactions.",
+  "smart-contracts-1-b-explanation": "Smart contracts can actually execute transactions automatically, but only according to an ‘if this then that’ logic that is prescribed in the code. They do not use AI.",
+  "smart-contracts-1-c-label": "They are programs stored on the blockchain that follow an 'if this then that' logic, and are guaranteed to execute according to the rules defined by the code.",
+  "smart-contracts-1-c-explanation": "A smart contract is a piece of code that can be executed automatically according to an 'if this then that' logic.",
+  "smart-contracts-1-d-label": "They are the rules behind the Ethereum Blockchain, developed together with lawyers to ensure legal compliance.",
+  "smart-contracts-1-d-explanation": "Smart contracts are pieces of code that can be created by developers and deployed on a blockchain.",
+  "smart-contracts-2-prompt": "What is a metaphor that most closely characterizes the functioning of smart contracts?",
+  "smart-contracts-2-a-label": "A bank",
+  "smart-contracts-2-a-explanation": "Banks require manual intervention and are structured as hierarchical entities, whereas a smart contract automatically executes without intervention.",
```

Again expressing caution around using "automatic" for smart contracts... They still must be told to "start" via a signed message and providing ETH as gas (computation fee)... it's the rest of the execution that is automatic:

```suggestion
  "smart-contracts-2-a-explanation": "Banks require manual execution and are structured as hierarchical entities, whereas smart contracts are executed predictable by computers with unchangeable rules.",
```